hack/setup.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+
+# Exit immediately if a command exits with a non-zero status.
+set -e
+
+echo "Installing Knative Serving Custom Resources..."
+kubectl apply -f https://github.com/knative/serving/releases/download/knative-v1.12.3/serving-crds.yaml
+
+echo "Installing Knative Serving Core Components..."
+kubectl apply -f https://github.com/knative/serving/releases/download/knative-v1.12.3/serving-core.yaml
+
+echo "Installing Knative Kourier Network Layer..."
+kubectl apply -f https://github.com/knative/net-kourier/releases/download/knative-v1.12.3/kourier.yaml
+
+echo "Configuring Knative Serving to use Kourier..."
+kubectl patch configmap/config-network \
+  --namespace knative-serving \
+  --type merge \
+  --patch '{"data":{"ingress-class":"kourier.ingress.networking.knative.dev"}}'
+
+echo "Fetching External IP or CNAME..."
+kubectl --namespace kourier-system get service kourier
```

Comment:
I don't think we actually need this, this is just verifying that kourier is running, but we don't check it anywhere. Maybe we can remove it for now?